src/tests/SConscript

```diff
@@ -31,6 +32,13 @@ def scons():
                                  LIBS=libs)
     denv.Install('$PREFIX/bin/', obj_ctl)
 
+    # install yamls files
+    env.Install('$PREFIX/etc/daos', 'ftest/data/daos_server_baseline.yaml')
+
+    # copy src/tests/ftest to $PREFIX/lib/daos/TESTING/ftest
+    spath = os.path.join(Dir('ftest').srcnode())
+    denv.InstallAs('$PREFIX/lib/daos/TESTING/ftest/', spath)
```

Comment:
lib64?

Reply:
It's actually /lib, not /lib64. We are moveing ftest to lib/daos/TESTING after moving all libraries under /lib to /lib64, this way, it's more in sync with RPM package code structure.